### ISSUE TITLE
question: also accept emerald ore from gems and jewels mod

### ISSUE THREAD

**Issue by Klausthorn** [kwpugh/gobber#83]
I tried to craft the ring of the miner with Gobber 2.3.41. But it did not work because the emerald ore block is not from minecraft but from https://www.curseforge.com/minecraft/mc-mods/gems-jewels ("gemsnjewels:emerald_ore_block"). Can you accept that block, too? 

Or better, any "*:*emerald_ore*" without specifying each existing mod's one? 

Isn't this a common problem when using more than one mod that has been solved?

Thanks for the fun mod!

Klaus

**Comment by kwpugh** [comment]
Hi,

Since the emerald ore from that mod spawns in many biomes, it would defeat the purpose of having a more rare vanilla Minecraft emerald ore in the Ring of the Miner recipe.

So no, I will not be adding tags to the recipe, I would like to keep the item challenging to obtain.

**Comment by Klausthorn** [comment]
good point. I did not think of that. On the other hand, I change the difficulty by adding mods all the time. That is part of the process of choosing mods.

And the way it looks now in the crafting UI is confusing for players and looks like a bug. Name and picture of the ore block are identical.

What I also noticed was that when I finally had a pick with silk touch and found 2 emerald ore and thus finally all I needed for the ring of the miner, I had mined so much as a byproduct that I did not need the ring any more. (At least in the overworld. It did still helped me to find Ancient Debris.) So this ring is at least not what I had hoped for in a mod.
